### PR TITLE
Docs: Remove `translog.flush_threshold_ops` table setting.

### PR DESCRIPTION
**Pull request by matriv**
This setting has been removed since ES5 upgrade.

### CODE REVIEW

File: blackbox/docs/sql/statements/create-table.rst [hunk 1]
```diff
@@ -519,4 +511,4 @@ the NGramTokenizer or the NGramTokenFilter. The default is 1.
 ....................
 
 Specifies the maximum difference between min_shingle_size and max_shingle_size
-when using the ShingleTokenFilter. The default is 3.
\ No newline at end of file
+when using the ShingleTokenFilter. The default is 3.
```
Comment (mxm) [inline blackbox/docs/sql/statements/create-table.rst]: Unrelated change 🤓

Comment (matriv) [reply blackbox/docs/sql/statements/create-table.rst]: it's some automatic trimming of trailing whitespace :-D

**Comment by matriv** [comment]
Fixes: https://github.com/crate/crate/issues/7242

**Comment by matriv** [comment]
Backported to 2.3 with: https://github.com/crate/crate/commit/79c097a8f7acf5795e5ae3d7e3ff065c5507c8ac
